src/mesh/Draco_Mesh.hh

```diff
@@ -134,6 +137,12 @@ public:
   const std::vector<unsigned> &get_cell_to_node_linkage() const {
     return m_cell_to_node_linkage;
   }
+  const std::vector<unsigned> &get_side_node_count() const {
```

Comment:
You are returning a reference to member data. If the class goes out of scope the reference will be invalidated.  Is this the design you want?  (It is fine -- as long as you understand the possible downside).

Reply:
Thank you for the great question - so far the data has only been accessed when the class is in scope.  When the mesh is in scope, it may be expedient to access by reference.  But so far, when these vectors are accessed they're usual manipulated eventually and copied to another data structure.